airbyte-server/src/test/java/io/airbyte/server/handlers/DestinationDefinitionsHandlerTest.java

```diff
@@ -146,6 +151,11 @@ void testGetDestination() throws JsonValidationException, ConfigNotFoundExceptio
   void testCreateDestinationDefinition() throws URISyntaxException, IOException, JsonValidationException {
     final StandardDestinationDefinition destination = generateDestination();
     when(uuidSupplier.get()).thenReturn(destination.getDestinationDefinitionId());
+    when(specFetcher.executeWithResponse(destination.getDockerRepository() + ":" + destination.getDockerImageTag()))
+            .thenReturn((SynchronousResponse<ConnectorSpecification>) jobResponse);
```

Comment:
i don't think this cast should be necessary. if you check in intellij i think it'll be grayed out.

Reply:
Fixed